public/app/features/admin/UserOrgs.tsx

```diff
@@ -242,7 +245,7 @@ class UnThemedOrgRow extends PureComponent<OrgRowProps> {
               onCancel={this.onCancelClick}
               onConfirm={this.onOrgRemove}
             >
-              Remove from organization
+              {t('admin.user-orgs.remove-button', 'Remove from organization')}
```

Comment:
note: we **have** to use `t` here. if we use `<Trans>`, `<ConfirmButton>` thinks the child is a button, tries to pass a load of things to it and ultimately breaks all the styling 🤮